src/content/docs/usage/targets.mdx

```diff
@@ -0,0 +1,313 @@
+
+## Create a Target
+
+Daytona provides an option to create a Target. Creating a Target requires creating a Target Configuration and specifying the Provider to use.
+
+1. Run the following command to create a Target:
+
+```shell
+daytona target create
+```
+
+:::note
+If you have not previously installed a Provider, registered a Runner, or created a Target Configuration, Dayona will prompt you to complete these steps before creating a Target.
+:::
+
+<br />
+
+2. Select the appropriate Provider for the environment you want to deploy to.
+
+```text
+Choose a provider
+4 items
+===
+Docker
+v0.0.0
+DigitalOcean
+v0.0.0
+AWS
+v0.0.0
+Azure
+v0.0.0
+GCP
+v0.0.0
+Hetzner
+v0.0.0
+Fly
+v0.0.0
+```
+
+:::note
+If you have not previously installed a [Runner](/docs/usage/runners), Daytona will prompt you select a Runner to manage the Provider. Upon selecting, Daytona will install the Runner on the target system and connect it to the Daytona Server.
+:::
+
+<br />
+
+3. Enter a name for your Target Configuration.
+
+```text
+Target Config Name
+>
+```
+
+4. Enter the appropriate configuration options when prompted. The configuration options vary based on the selected Provider. The following example shows adding a local Docker Target Configuration.
+
+```text
+Remote Hostname
+>
+
+Remote Password
+>
+
+Remote Port
+>
+
+Remote User
+Note: non-root user required
+>
+
+Sock Path
+> /var/run/docker.sock
+
+Workspace Data Dir       
+The directory on the remote host where the workspace data will be stored
+> /tmp/daytona-data
+
+Remote Private Key Path
+daytona_config
+known_hosts
+Custom path
+None
+```
+
+5. Enter a name for your Target.
+
+```text
+Target Name
+>
+```
+
+```text
+✓ Request submitted
+Creating target <target_name>
+```
+
+## Start a Target
+
+Daytona provides an option to start a Target.
+
+1. Run the following command to start a Target:
+
+```shell
+daytona target start
+```
+
+2. Select the Target you want to start.
+
+```text
+Select Targets to Start
+===
+local (default)
+<target_id>
+Docker (local)
+```
+
+## Stop a Target
+
+Daytona provides an option to stop a Target.
+
+1. Run the following command to stop a Target:
+
+```shell
+daytona target stop
+```
+
+2. Select the Target you want to stop.
+
+```text
+Select Targets to Stop
+===
+local (default)
+<target_id>
+Docker (local)
+```
+
+## List Targets
+
+Daytona provides an option to list all Targets.
+
+1. Run the following command to list all Targets:
+
+```shell
+daytona target list
+```
+
+Upon running the command, Daytona will display a list of all Targets along with their configurations.
+
+```text
+Target          Config Name     # Workspaces    Default     Status
+local (Docker)  local           0               Yes         Running
```

Comment:
Docker targets don't have state so their `Status` would be `/`

You can put an example of an AWS target here that will have the Status as `Started`